image_width: 1280
image_height: 720
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [590.7465 ,   0.     , 635.42353,
           0.     , 590.75577, 410.56854,
           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.029408, -0.012060, -0.000370, -0.000097, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [554.34119,   0.     , 633.68064,   0.     ,
           0.     , 581.75757, 411.27716,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]